src/main/java/uk/nhs/digital/nhsconnect/nhais/model/jsonpatch/AmendmentPatch.java

```diff
@@ -17,6 +17,7 @@
 public class AmendmentPatch {
 
     private static final String REMOVE_INDICATOR = "%";
+    private static final String EMPTY_STRING = "";
```

Comment:
StringUtils.EMPTY from commons-lang3 is what we're using in other parts of the application